public/css/main.css

```diff
@@ -179,23 +179,51 @@ footer {
 
 .grayButton {
     position: relative;
-    background-color: #eee;
+    background-color: rgba(251, 215, 139, 1);
     -moz-border-radius:15px;
     -webkit-border-radius:15px;
     border-radius:10px;
-    border:1px solid #eee;
+    border:1px solid rgba(251, 215, 139, 1);
     display:inline-block;
     position: relative;
     top:8px;
     cursor:pointer;
     color:#2D2A3F;
     font: normal normal normal 14px/1.4em raleway,sans-serif;
     font-family: 'Raleway', sans-serif;
+    text-align: start;
+    font-weight: bolder;
     font-size:14px;
     padding:5px 25px;
     text-decoration:none;
     transition: border-color 0.4s ease 0s, background-color 0.4s ease 0s;
 }
+
+.grayButtonMobile {
```

Comment:
Is this used anywhere?

Reply:
Yes it's used for the sign in button on mobile, it's code from my mobile sign in page pr ad it shouldn't affect anything.

Reply:
Okay, it should not be a part of this PR though. Could you go through and remove this and anything else remaining that is not meant to be part of the navbar PR

Reply:
Done, that should be the only thing because none of my other PRs touched main.css or navbar.scala.html